cl/api/urls.py

```diff
@@ -152,6 +153,11 @@
         include("rest_framework.urls", namespace="rest_framework"),
     ),
     re_path(r"^api/rest/(?P<version>[v3]+)/", include(router.urls)),
+    re_path(
+        r"^api/rest/(?P<version>[v3]+)/citation-lookup/",
+        citations_views.CitationLookupView.as_view(),
+        name="citation_lookup",
+    ),
```

Comment:
Did you try registering this endpoint, like the examples above?

Reply:
No, I didn't because the `CitationLookupView` inherits from `GenericAPIView`, which isn't compatible with the default router. However, updating the view to work with the router is a straightforward process.